project/Dependencies.scala

```diff
@@ -58,6 +59,7 @@ object Dependencies {
   val logback = "ch.qos.logback" % "logback-classic" % Version.logback
   val wireMock = "com.github.tomakehurst" % "wiremock" % Version.wireMock % Test
   val sl4j = "org.slf4j" % "slf4j-api" % Version.sl4j
+  val slf4j = "org.slf4j" % "slf4j-api" % Version.slf4j
```

Comment:
Why do we need the same dependency twice?

Reply:
Fix it